config/application.rb

```diff
@@ -47,6 +47,7 @@ class Application < Rails::Application
 
     config.active_support.to_time_preserves_timezone = :zone
 
-    Time::DATE_FORMATS[:date_only] = "%b %d, %Y"
+    Time::DATE_FORMATS[:date_only] = "%-d %B %Y"
+    Date::DATE_FORMATS[:date_only] = "%-d %B %Y"
```

Comment:
Nice refactor!
but `:date_only` does not say much about the date,
we can have date only in many different format.
Should we rename it to something like:
`:day_month_year`
`:readable_date`
`:long_date`
...
up to you :)